tests/commons/yml/test_load_resolve.py

```
@@ -0,0 +1,34 @@
+
+def test_resolve_loader() -> None:
+    context = {"a": "1", "b": "2"}
+
+    yml_text = "value: !resolve '{a}.{b}'"
+    # 0. with std resolver we get an error for not ignoring unknown tags
+    with pytest.raises(yaml.constructor.ConstructorError):
+        out = yaml.load(yml_text, Loader=yaml.SafeLoader)
+
+    # 1. with ignore_unknown resolver we get out without expansion applied
+    ignoring_loader = LoaderBuilder().ignore_unknown().build()
+    out = yaml.load(yml_text, Loader=ignoring_loader)
```

Ruff (0.8.2)

Probable use of unsafe loader `ignoring_loader` with `yaml.load`. Allows instantiation of arbitrary objects. Consider `yaml.safe_load`.

(S506)